src/test/playwright/support/pageobjects/exam/ExamManagementPage.ts

```diff
@@ -101,11 +101,11 @@ export class ExamManagementPage {
         await expect(this.page.locator('#student-exam .datatable-body-row', { hasText: username }).locator('.submitted')).toHaveText('Yes');
     }
 
-    async checkQuizSubmission(courseID: number, examID: number, username: string, score: string) {
+    async checkQuizSubmission(courseID: number, examID: number, username: string, score1: string) {
```

Comment:
Out of curiosity - why `score1` instead of `score`? 😄

Reply:
i forgot to revert the change ups. I first had an assertion of 2 partial strings thats why score 1 and score 2 😄